draft-ietf-tls-tls13.md

```diff
@@ -3340,14 +3340,14 @@ provides several recommendations to assist implementors.
 
 ## Random Number Generation and Seeding
 
-TLS requires a cryptographically secure pseudorandom number generator (PRNG).
+TLS requires a cryptographically secure pseudo-random number generator (PRNG).
```

Comment:
This doesn't need a hyphen either.
https://en.wikipedia.org/wiki/Pseudorandomness